Wikipedia/Code/WMFSettingsMenuItem.m

```diff
@@ -136,6 +136,16 @@ + (WMFSettingsMenuItem *)itemForType:(WMFSettingsMenuItemType)type {
                                        disclosureText:nil
                                            isSwitchOn:[SessionSingleton sharedInstance].dataStore.readingListsController.isSyncEnabled];
         }
+        case WMFSettingsMenuItemType_StorageAndSyncingDebug: {
+            return
+            [[WMFSettingsMenuItem alloc] initWithType:type
+                                                title:@"Reading list danger zone"
```

Comment:
Looks like we need a localized string here.

Reply:
This is only for debugging though

Reply:
Haha you're right! I just saw a string and didn't even read it :-P

Reply:
![simulator screen shot - iphone x - 2018-02-21 at 07 45 52](https://user-images.githubusercontent.com/741327/36480766-80a6d75c-16db-11e8-9a27-21a59863eea1.png)